packages/cactus-plugin-ledger-connector-xdai/src/test/typescript/integration/deploy-contract-from-json-xdai.test.ts

```diff
@@ -252,11 +252,9 @@ describe(testCase, () => {
           type: Web3SigningCredentialType.PrivateKeyHex,
         },
         nonce: 1,
-      });
-      fail("It should not reach here");
-    } catch (error) {
-      expect(error).not.toBe("Nonce too low");
-    }
+      }),
+    ).rejects.toThrow();
```

Comment:
Can you verify the error thrown here? It's possible that the original check was wrong here, but we could still verify that the error is truely expected here